### PR TITLE
Added testcast to validate existing fix (MLDB-815)

### PR DESCRIPTION
**Pull request by jeremybarnes**
Port of a stash of a (now passing) testcase.  No functional changes.

### CODE REVIEW

**Comment by FinchPowers** [comment]
+1

File: testing/testing.mk [hunk 1]
```diff
@@ -320,8 +320,7 @@ $(eval $(call mldb_unit_test,MLDBFB-192_row_name_as_string_test.py))
 $(eval $(call mldb_unit_test,MLDBFB-199_invalid_script_test.py))
 $(eval $(call mldb_unit_test,MLDBFB-208_procedure_params_overwrite_test.py))
 $(eval $(call mldb_unit_test,MLDB-687-svd-embed-row-single-column.js))
-$(eval $(call mldb_unit_test,get_http_bound_address.py))
-$(eval $(call mldb_unit_test,get_http_bound_address.js))
 $(eval $(call mldb_unit_test,MLDB-1317-tensor-datatype.js))
 $(eval $(call mldb_unit_test,get_http_bound_address.py))
 $(eval $(call mldb_unit_test,get_http_bound_address.js))
+$(eval $(call mldb_unit_test,MLDB-815-sparse-mutable-record-strings.js))
```
Comment (mailletf) [inline testing/testing.mk]: should be manual if it's going to fail

**Comment by mailletf** [comment]
+1

**Comment by FinchPowers** [comment]
= 2

**Pull request merged**